documentation/src/main/markdown/releasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Release notes
 
+## Version 8.9.0
+
+### Resolved issues
+
+* (IDETECT-3613) When running a scan with `detect.maven.build.command=-Dverbose`, resolved KB mismatch issue for omitted transitive dependencies.
```

Comment:
New file location for this will be: _currentreleasenotes.md_
Suggested edit:
(IDETECT-3613) Resolved an issue where running a scan with `detect.maven.build.command=-Dverbose` caused a KB mismatch issue for omitted transitive dependencies.